tests/integration/test_ses.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+from datetime import date, datetime
+from localstack.utils.aws import aws_stack
+
+TEST_TEMPLATE_ATTRIBUTES = {
+    'TemplateName': 'hello-world',
+    'SubjectPart': 'Subject test',
+    'TextPart': 'hello\nworld',
+    'HtmlPart': 'hello<br/>world',
+}
+
+
+class SESTest(unittest.TestCase):
+
+    def test_list_templates(self):
+        client = aws_stack.connect_to_service('ses')
+        client.create_template(Template=TEST_TEMPLATE_ATTRIBUTES)
+        templ_list = client.list_templates()['TemplatesMetadata']
+        self.assertEqual(1, len(templ_list))
+        created_template = templ_list[0]
+        self.assertEqual(
+            created_template['Name'], TEST_TEMPLATE_ATTRIBUTES['TemplateName'])
+        self.assertTrue(
```

Comment:
Looks like the tests are failing on this assertion. I think we could change this to something like:
```
self.assertIn(type(created_template['CreatedTimestamp']), (date, datetime))
```